.eslintrc.js

```diff
@@ -19,7 +19,14 @@ module.exports = {
       '@typescript-eslint/no-require-imports': 'off',
       '@typescript-eslint/no-var-requires': 'off',
     },
-  }],
+  },
+  {
+    files: ['dependencies.d.ts'],
+    rules: {
+      'import/unambiguous': 'off',
+    },
+  },
+  ],
```

Comment:
This override will need to be changed once the `types` directory is created, and also, we should run `prettier --write --single-quote` on this file to fix the formatting. The indentation is a bit wonky at the moment.